core/src/main/java/tc/oc/pgm/filters/FilterParser.java

```diff
@@ -443,4 +444,30 @@ public StructuralLoadFilter parseStructuralLoad(Element el) throws InvalidXMLExc
   public TimeFilter parseTimeFilter(Element el) throws InvalidXMLException {
     return new TimeFilter(XMLUtils.parseDuration(el, null));
   }
+
+  @MethodParser("players")
+  public PlayerCountFilter parsePlayerCountFilter(Element el) throws InvalidXMLException {
+    Filter countFilter = parseFilterProperty(el, "count-filter", StaticFilter.ALLOW);
+
+    // TODO: Legacy syntax, maybe bump proto to deprecate
+    boolean participants = XMLUtils.parseBoolean(el.getAttribute("participants"), true);
+    boolean observers = XMLUtils.parseBoolean(el.getAttribute("observers"), false);
+    countFilter =
```

Comment:
use `ParticipatingFilter.PARTICIPATING` or `ParticipatingFilter.OBSERVING` instead of this annonymous filter